tests/testthat/test_iov.R

```diff
@@ -7,7 +7,21 @@ pars <- list(
   "V" = 50,
   "KA" = 1
 )
-
+pars0 <- list(
+  "CL" = 5,
+  "V" = 50,
+  "KA" = 1
+)
+pk0 <- new_ode_model( # no IOV
```

Comment:
calling new_ode_model is slow, and I don't think we strictly need this model for this test. we load a number of pretty basic models in our testing suite, can we use one of those to keep test time down?

Reply:
Yes, I tried, but couldn't find a good one while having the test be is in a logical test file.
 
What I could do is make a separate ticket to restructure the tests to use only the minimal number of calls to new_ode_model(), I think there is some unnecessary duplication going on as well.